src/wp-includes/fonts.php

```diff
@@ -13,13 +13,13 @@
  * @since 6.4.0
  *
  * @param array[][] $fonts {
- *     Optional. The font-families and their font variations. Default empty array.
+ *     Optional. The font-families and their font faces. Default empty array.
  *
- *     @type string $font-family => array[] $variations {
- *         Optional. An associated array of font variations for this font-family.
- *         Each variation has the following structure.
+ *     @type array {
+ *         Optional. An indexed array of font variations for this font-family.
```

Comment:
Here, the key is optional. It can be an associative or indexed array.

```suggestion
 *     @type array {
 *         An indexed or associative (keyed by font-family) array of font variations for this font-family.
```